src/Core/src/Platform/Android/NativeTicker.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using Android.Animation;
+using Android.Content;
+using Android.OS;
+using Microsoft.Maui;
+namespace Microsoft.Maui.Animations
+{
+	public class NativeTicker : Ticker
+	{
+		static List<NativeTicker> tickers = new List<NativeTicker>();
+		public static void CheckPowerSaveModeStatus()
+		{
+			foreach (var t in tickers)
+				t.checkPowerSaveModeStatus();
+		}
+		readonly IMauiContext MauiContext;
+		ValueAnimator _val;
+		bool _systemEnabled;
+		public NativeTicker(IMauiContext mauiContext)
+		{
+			MauiContext = mauiContext;
+			_val = new ValueAnimator();
+			_val.SetIntValues(0, 100); // avoid crash
+			_val.RepeatCount = ValueAnimator.Infinite;
+			_val.Update += (s, e) => Fire?.Invoke();
+			;
+			checkPowerSaveModeStatus();
+			tickers.Add(this);
+		}
+		~NativeTicker()
+		{
+			tickers.Remove(this);
+		}
+
+		internal void checkPowerSaveModeStatus()
+		{
+			// Android disables animations when it's in power save mode
+			// So we need to keep track of whether we're in that mode and handle animations accordingly
+			// We can't just check ValueAnimator.AreAnimationsEnabled() because there's no event for that, and it's
+			// only supported on API >= 26
+
+			//if (!Forms.IsLollipopOrNewer)
```

Comment:
Hooray, we don't have to worry about pre-Lollipop anymore! Can we just delete this section?

Reply:
Deleting!